### PR TITLE
Add basic configuration for GitHub Actions continuous integration

### PR DESCRIPTION
**Pull request by aforward**
Grant's PR, I am just seeing the permissions.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+name: Playground Push Action
+on: [push]
+jobs:
+  Continuous-Integration:
+    runs-on: ubuntu-latest
```
Comment (aforward) [inline .github/workflows/ci.yml:5]: Thoughts on a smaller default like `alpine` linux

**Review by aforward — COMMENTED** [review]
:+1: this was really grants PR

File: .github/workflows/ci.yml (hunk 1 above)
Comment (bbranan) [reply .github/workflows/ci.yml:5]: Perhaps not relevant for the playground, but for PASS components we may want to consider the expected deployment environment and try to match that relatively closely in the CI test environment.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+name: Playground Push Action
+on: [push]
```
Comment (bbranan) [inline .github/workflows/ci.yml:2]: It's often valuable to test on PRs as well
```suggestion
on: [push, pull_request]
```

File: .github/workflows/ci.yml (hunk 1 above)
Comment (grant-mcs) [reply .github/workflows/ci.yml:5]: This is the default Linux offered by GitHub. We will be using a self-hosted runner or a Docker image running on  a GitHub-hosted runner when up and running. This is where we will make a decision  based on what best matches the environment that each PASS component is targeted to. We should be configuring  the deployment targets and Actions runner using the same process.

File: .github/workflows/ci.yml (hunk 2 above)
Comment (grant-mcs) [reply .github/workflows/ci.yml:2]: Yes, absolutely.  I was just trying to confirm permissions with this initial commit. Once I get the permissions sorted out there will be many changes.

**Review by grant-mcs — APPROVED** [review]

**Pull request merged**